models.go

```diff
@@ -70,3 +71,10 @@ func (source *Source) GetPipelineName() string {
 	}
 
 }
+
+func (source *Source) GetSort() string {
+	if source.Sort != "" {
```

Comment:
@romain-dartigues can you accept only one of the supported value (`asc` or `desc`) to make it more robust?